core/workspace_svg.js

```diff
@@ -388,7 +388,7 @@ Blockly.WorkspaceSvg.prototype.createDom = function(opt_backgroundClass) {
     bottom = this.addTrashcan_(bottom);
   }
   if (this.options.zoomOptions && this.options.zoomOptions.controls) {
-    bottom = this.addZoomControls_(bottom);
+    this.addZoomControls_(bottom);
```

Comment:
why did this change?

Reply:
The `no-unused-variables` triggered on this: "Bottom is assigned a value but never used."

These functions take in a bottom position and return the top of the object after it has been laid out.  That can then be used to position another icon above it.  In this case, we're not positioning any other icons above.

Reply:
Ah, I see. I was thrown by the assignment on 388.  But makes sense.